### PR TITLE
Enable the use of external secretsservice.

### PR DESCRIPTION
**Pull request by jbygdell**
Some hacking where required to make this work, mainly in DOA and orchestrator.

### CODE REVIEW

**Review by blankdots — APPROVED** [review]
looks good. My only comments are about the names
`.values.global.vaultSecrets` would be cleared and the same for the `VaultConfigFile`

**Review by blankdots — COMMENTED** [review]
maybe also some changes in readme and version bump in charts would be nice

**Review by blankdots — APPROVED** [review]

File: sda-svc/templates/doa-deploy.yaml [hunk 1]
```diff
@@ -50,6 +50,10 @@ spec:
 {{- if .Values.global.extraSecurityContext }}
 {{- toYaml .Values.global.extraSecurityContext | nindent 10 -}}
 {{- end }}
+      {{- if .Values.global.vaultSecrets }}
+        command: ["/bin/sh"]
+        args: ["-c", "source {{ include "secretsPath" . }}/{{.Values.global.confFile}} && docker-entrypoint.sh && java -jar /localega-doa.jar"]
```
Comment (pontus) [inline sda-svc/templates/doa-deploy.yaml:55]: I'm not sure I manage to keep track, so just to check - is this really sourcable?

Comment (jbygdell) [reply sda-svc/templates/doa-deploy.yaml:55]: It all depends on how the file created by Vault looks like, but for DOA to start the file should look like this:

```
export BROKER_USER=<username>
export BROKER_PASSWORD=<password>

```

Comment (jbygdell) [reply sda-svc/templates/doa-deploy.yaml]: Made it a bit clearer for the DOA case

**Review by pontus — APPROVED** [review]

**Pull request merged**